### ISSUE TITLE
This extension is the subject of an attack vulnerability

### ISSUE THREAD

**Issue by pbolger** [dadmor/ACF_frontend_display#12]
92.63.87.61 - - [26/Oct/2015:19:55:47 +1300] "POST /wp-content/plugins/acf-frontend-display/js/blueimp-jQuery-File-Upload-d45deb1/server/php/index.php HTTP/1.1" 404 566 "-" "Mozilla/5.0 (Windows NT 6.1; rv:34.0) Gecko/20100101 Firefox/34.0"

https://www.exploit-db.com/exploits/37514/

**Comment by dadmor** [comment]
Yes - now wrong code is removed - in future I'll rebuild this plugin becouse now I have many different goals with my authonomic form system:
http://uigen.org/jsonforms/
